CONTRIBUTING.md

```diff
@@ -6,7 +6,17 @@ Start by opening an issue using one of the issue templates, or propose a change
 
 ## Running the Project
 
-### Build Gradle Server and Gradle Language Server.
+### Build Gradle Project Importer
+Before proceeding with the build steps for Gradle Server , you need to build the Gradle Project Importer first.
```

Comment:
Before..., you need to build the Gradle Build Server and its client (Gradle Project Importer) first.